src/global/styles/normalize.scss

```diff
@@ -230,3 +230,46 @@ Add the correct display in Chrome and Safari.
 summary {
   display: list-item;
 }
+
+/*
```

Comment:
Probably, this is not the correct place where this CSS should be.
These rules achieve the "wrapping" effect of the overlay on the origin element